modules/authentication/reducer.js

```diff
@@ -51,7 +51,8 @@ function authentication(state = INITIAL_STATE, action) {
     return {
       ...state,
       error: null,
-      isActive: true
+      isActive: true,
+      tokenInfo: {}
```

Comment:
Well... I can't recreate this error so I can't tell you why it was being passed down. I think we're good.